src/components/paradox/results/personality/career/details/index.js

```diff
@@ -63,6 +64,13 @@ function CareerDetails(props) {
               <button type="button" onClick={openModal}>Learn More</button>
             </div>
           </div>
+          {career?.inline_jobs && (
```

Comment:
It looks like you don't need the `?` since other places in this component it's relying on `career` existing